samples: mesh/onoff_level_lighting_vnd_app: States binding corrections

vikrant8052 opened the pull request:
Make corrections in state binding of Servers as per Mesh Model
Specifications. Previously, when OnPowerUp state equal to 0x02,
then Light Lightness actual state was not assign to last power
down value. Plus when Generic OnOff state changes by client,
then Light Lightness Actual state value get assigned as Light
Lightness Last value instead of default one.
All these issues has fixed here.

Signed-off-by: Vikrant More <vikrant8051@gmail.com>

vikrant8051 commented:
@jhedberg Accidentally I've closed down my old PR & couldn't open it. Since I've executed git push -f after that. Please bare me for this time. 

Please ask @carlescufi & @sjanc to review this new PR. Currently busy in other tasks & can't concentrate on all this. But I can't let others to use an app with bugs.

I'm now sure that there is no single bug in App.

vikrant8052 committed:
samples: mesh/onoff_level_lighting_vnd_app: States binding corrections

Make corrections in state binding of Servers as per Mesh Model
Specifications. Previously, when OnPowerUp state equal to 0x02,
then Light Lightness actual state was not assign to last power
down value. Plus when Generic OnOff state changes by client,
then Light Lightness Actual state value get assigned as Light
Lightness Last value instead of default one.
All these issues has fixed here.

Signed-off-by: Vikrant More <vikrant8051@gmail.com>

vikrant8051 commented:
@carlescufi @jhedberg @sjanc  Please approve this asap, so that others can test actual app without any inherent bugs.
Thanks !!

vikrant8051 commented:
@jhedberg Thank You for merging it.
